packages/repository/src/__tests__/fixtures/mixins/findByTitleInterface.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
```

Comment:
Please rename this file to `find-by-title-interface.ts`.

Personally, I find it too verbose to have a dedicated file for this interface, have you considering moving the interface to the same file where `FindByTitleRepositoryMixin` is defined?